codexdht/private/eth/p2p/discoveryv5/transport.nim

```diff
@@ -93,7 +93,7 @@ proc sendMessage*(t: Transport, toNode: Node, message: seq[byte]) =
       t.send(toNode, data)
       t.keyexchangeInProgress.incl(toNode.id)
       trace "keyexchangeInProgress added", myport = t.bindAddress.port , dstId = toNode
-      sleepAsync(responseTimeout).addCallback() do(data: pointer):
+      sleepAsync(handshakeTimeout).addCallback() do(data: pointer):
```

Comment:
Since we're in proc sendMessage it's odd that we're using handshake timeout instead of response timeout. Maybe the else: regarding the handshake process can be its own proc with its own name?

Reply:
We are in sendMessage, but we are on an else branch, starting a keyexchage, as you've also noted.
I agree it is a bit confusing, and a more general review of timeouts is needed, including some code factorization. Lets do that in a separate PR, while fix the used value here.